src/projects/forms.py

```
@@ -6,9 +6,45 @@
+                self.initial['scheme'] = 'http'
+                self.initial['url'] = self.instance.url.replace('http://', '')
+
+    def clean_url(self):
+        url = self.cleaned_data['url']
+        if not url.startswith('http://') and not url.startswith('https://'):
+            raise ValidationError('URL must start with http:// or https://')
+        return url
+    
+    def save(self, commit=True, user=None):
+        url = self.cleaned_data['url'].replace('https://', '').replace('http://', '')
+        domain = url
```

Ruff

Local variable `domain` is assigned to but never used

Remove assignment to unused variable `domain`

(F841)